eureka/S6_planet_spectra/plots_s6.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+def plot_spectrum(meta, wavelength, spectrum, err, wavelength_error=None, model_x=None, model_y=None, ylabel=r'$R_{\rm p}/R_{\rm *}$', xlabel=r'Wavelength ($\mu$m)'):
```

Comment:
A nice option to have is to include scale heights on the right axis. Perhaps we make this a separate figure with isplots=3?

Reply:
Good thinking - I'll reuse this function and just call it twice if isplots==3 with a boolean flipped

Reply:
But only for transits!

Reply:
How do you want to convert between radii and H? Do you want people to pass in the scale height, pass in some of the values and look up the rest from the exoplanet archive?